stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1597 @@
+{
```

Comment:
<p>Not sure if this is worth exploring, but any interest in an increased supply scenario? For example, the increase in doses that started mid-March (aside from the Easter dip) seems to be a higher level compared to the average from the beginning of the year to mid-March. What if the projected rate was based on the average of the previous three months?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>I'm reluctant to make the story longer by adding another forecast, but I've gone back and forward a lot about modifying one of the pre-existing ones. In the end I realised using the week up to 4th May includes a bank holiday, so I've changed the first forecast to use the average from the last 2 weeks to be more representative of the current situation (which changes the end date of the first forecast to 26th August instead of 31st August). But I also think it's likely that rates will ultimately fall near the end of the programme, once the remaining population that hasn't been vaccinated consists mostly of people who are at low risk from Covid-19 (i.e. young people) or are at risk but reluctant to be vaccinated. I've added a sentence or two about that as well.</p>